FEXCore/Source/Interface/Core/JIT/JITClass.h

```diff
@@ -61,6 +62,15 @@ class Arm64JITCore final : public CPUBackend, public Arm64Emitter {
   const bool HostSupportsRPRES {};
   const bool HostSupportsAFP {};
 
+  struct RestartOptions {
+    FEXCore::LongJump::JumpBuf RestartJump;
+    bool NeedsLongJumps {};
```

Comment:
Should this variable really be part of RestartOptions? IIUC it's just a general behavior toggle of the JIT, so perhaps it's better placed in Arm64JitCode itself? I found it a bit surprising to read `RestartControl.NeedsLongJumps` for code that wasn't itself related to the JIT restarts at all.